.gitignore

```diff
@@ -19,3 +19,4 @@ yarn-debug.log*
 yarn-error.log*
 
 /temp
+package-lock.json
```

Comment:
It shouldn't exist if you know what the package manager prefers in this project; you know this from `package.json`'s `packageManager` field